### ISSUE TITLE
How to Stop audio stream from google to my program.

### ISSUE THREAD

**Issue by alimohyudin** [endoplasmic/google-assistant#27]
Hi, 
Everything is working fine. :) but I want to stop audio stream from google sometimes. 
For example:

1- I speak to assistant: Tell me something about guns.
2- Assistant sends me back transcription as "Tell me something about guns." with audio response (I don't want audio response)
3- I want my program to say: Are you sure you want to know about guns? (I can do it by using transcription, no problem)
4- Then I will perform next task I want to do.

On 2nd step, I just don't want my assistant to continue, I just want it to Stop Forcefully!
I can remove the audio-data event but the conversation is not properly end there. how I can achieve that?

Thanks in advance!

**Comment by endoplasmic** [comment]
This one is tricky, I tired to do this a while back without success. I'll keep an eye out to see if this ever gets supported though.

**Comment by alimohyudin** [comment]
Yes sure and it is tricky but it exists as I tested Google Assistant on Android and it doesn't give me audio back but just respond with an answer and the conversation continuous without audio from google. But I will let you know if I get any success so that it will be available for everyone.
